test/cmd/legacy-script.sh

```diff
@@ -377,7 +377,6 @@ runTests() {
   export container_name_field="(index .spec.template.spec.containers 0).name"
   export hpa_min_field=".spec.minReplicas"
   export hpa_max_field=".spec.maxReplicas"
-  export hpa_cpu_field=".spec.targetCPUUtilizationPercentage"
```

Comment:
what is the v2 equivalent of this? shouldn't we check for that rather than dropping this entirely?

Reply:
(and add this back to all the tests)